src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/SourceInfo.cs

```diff
@@ -1,15 +1,16 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
-using System.Collections.Generic;
 using System.Collections.Immutable;
 using Microsoft.CodeAnalysis.FlowAnalysis.DataFlow.PointsToAnalysis;
 using Microsoft.CodeAnalysis.FlowAnalysis.DataFlow.ValueContentAnalysis;
+using Microsoft.CodeAnalysis.Operations;
 
 namespace Analyzer.Utilities.FlowAnalysis.Analysis.TaintedDataAnalysis
 {
-    public delegate bool IsInvocationTaintedWithPointsToAnalysis(IEnumerable<PointsToAbstractValue> argumentPointsTo);
-    public delegate bool IsInvocationTaintedWithValueContentAnalysis(IEnumerable<PointsToAbstractValue> argumentPonitsTos, IEnumerable<ValueContentAbstractValue> argumentValueContents);
+    internal delegate bool PointsToCheck(ImmutableArray<PointsToAbstractValue> pointsTos);
+    internal delegate bool ValueContentCheck(ImmutableArray<PointsToAbstractValue> pointsTos, ImmutableArray<ValueContentAbstractValue> valueContents);
+    internal delegate bool MethodMapper(string methodName, ImmutableArray<IArgumentOperation> arguments);
```

Comment:
>MethodMapper [](start = 27, length = 12)

Maybe call this "MethodMatcher"?  Since it's not really mapping anything.  Well I guess it's mapping to true or false.  🙂